app/javascript/chat/channelDetails.jsx

```diff
@@ -144,16 +145,25 @@ class ChannelDetails extends Component {
           if (this.userInList(channel.channel_users, user)) {
             invite = (
               <span className="channel__member">
-                is already in <em>{channel.channel_name}</em>
+                is already in 
+                {' '}
+                <em>{channel.channel_name}</em>
               </span>
             );
           }
           return (
             <div className="channeldetails__searchedusers">
               <a href={user.path} target="_blank" rel="noopener noreferrer">
-                <img src={user.user.profile_image_90} alt="profile_image" />@
-                {user.user.username} - {user.title}
-              </a>{' '}
+                <img src={user.user.profile_image_90} alt="profile_image" />
+@
+                {user.user.username}
+                {' '}
+-
+                {/* prettier-ignore */} 
```

Comment:
Interestingly, Prettier always removes this whitespace which leads to a failing test.